### PR TITLE
GPU picker improvements

### PR DESCRIPTION
**Pull request by audionerd**

### CODE REVIEW

File: src/js/shot-generator/Character.js [hunk 1]
```diff
@@ -293,14 +294,17 @@ const Character = React.memo(({
           z : rotation.z,
         }  
       } );}); 
-      console.log(mesh.name);
+      console.log(mesh);
+      //TODO(): Make a better way to identify built in model
```
Comment (audionerd) [inline src/js/shot-generator/Character.js]: If isCustomModel returns false, it's a built-in model

File: src/js/shot-generator/SelectionManager.js [hunk 2]
```diff
@@ -32,6 +34,7 @@ function getObjectsFromIcons ( objects ) {
       )
 }
 
+// TODO() : Remove this
```
Comment (audionerd) [inline src/js/shot-generator/SelectionManager.js]: Why does this need to be removed?

File: src/js/shot-generator/Character.js (hunk 1 above)
Comment (VladStepura) [reply src/js/shot-generator/Character.js]: I'm not sure that isCustomModel returns correct answer. I'm loading horse.glb and it's showing that it's built-in. By looking into the logic of that method it just checks if path is not empty.

Comment (audionerd) [reply src/js/shot-generator/Character.js]: You'd pass it the `sceneObject.model`

**Pull request merged**